xcdat/spatial_avg.py

```diff
@@ -47,9 +49,10 @@ def avg(
 
         Parameters
         ----------
-        data_var_name: str
+        data_var: Optional[str], optional
             The name of the data variable inside the dataset to spatially
-            average.
+            average. If None, an inference to the desired data variable is
+            attempted using the Dataset's "xcdat_infer" attr, by default None.
```

Comment:
"...is attempted using ``get_inferred_var()`` and the Dataset's 'xcdat_infer' attr."